### DISCUSSION TITLE
I want to help with the documentation

### DISCUSSION THREAD

**Discussion by yudinmaxim** [open-circle/valibot#395] [Ideas]
I can translate the existing documentation into Russian and have tried to write the missing parts in Russian. My English is not sufficient for writing technical documents. I can write such a document in Russian. If it's possible, how can I do this?

**Comment by fabian-hiller** [comment]
Thank you for your offer! We do not have the setup yet to translate the documentation into other languages. I will work on it as soon as the library is more popular.

**Reply by yudinmaxim** [reply]
Thank you for you responce.
I hope you library get more popular soon.

**Reply by fabian-hiller** [reply]
I will be publishing our i18n feature as a draft PR in the next few hours. If you are interested, you can translate our internal error messages to Russian after I merge this PR into the main branch.
